pkgs/development/ocaml-modules/tezos/lwt-result-stdlib.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildDunePackage
+, ocaml
+, tezos-stdlib
+, tezos-error-monad
+, alcotest
+, alcotest-lwt
+, crowbar
+}:
+
+if lib.versionAtLeast ocaml.version "4.12" then
+  throw "tezos-lwt-result-stdlib-${tezos-stdlib.version} is not available for OCaml > 4.10"
+else
+
+buildDunePackage {
+  pname = "tezos-lwt-result-stdlib";
+  inherit (tezos-stdlib) version src useDune2 preBuild doCheck;
+
+  #maximumOCamlVersion
```

Comment:
What’s the purpose of this comment?

Reply:
Forgot to remove it, it was before I did the if/throw thing on line 10